code/obj/item/plants.dm

```diff
@@ -252,6 +251,12 @@
 	desc = "Chewy leaves often manufactured for use in radiation treatment medicine."
 	icon_state = "nureous"
 
+/obj/item/plant/herb/nureous/fuzzy
+	name = "nureous leaves"
+	crop_suffix	= " leaves"
```

Comment:
Weird spacing (between the `crop_suffix` and the `=`, I get the need for the one before `leaves`.)

Reply:
I don't see any issue here.

Reply:
Woah, on mobile I actually do. Check out the specific character used there/overwrite it

Reply:
I'll have a look now

Reply:
Weird, I deleted and retyped the space and it listed it as a delta, so i guess it was a weird character. Oh well, issue resolved in latest commit

Reply:
Resolved with b6cb1a4.